integration_tests/src/main/python/delta_lake_utils.py

```diff
@@ -31,6 +32,14 @@
     "SortExec"
 ]
 
+# Disable Deletion Vectors except for Databricks 14.3
+def deletion_vector_conf_with_reason(reason='https://github.com/NVIDIA/spark-rapids/issues/12042'):
+    return list(filter(None, ({'spark.databricks.delta.properties.defaults.enableDeletionVectors': 'false'},
+                              pytest.param({'spark.databricks.delta.properties.defaults.enableDeletionVectors': 'true'},
+                                           marks=pytest.mark.xfail(
+                                               reason=reason)) if is_databricks143_or_later() else None)))
+deletion_vector_conf = deletion_vector_conf_with_reason()
```

Comment:
nit: can we add an s to the end of deletion_vector_conf and make it clear that this is really only for writes? `deletion_vector_write_confs`, which makes it a little more clear that this is a list.

Reply:
Never mind I misunderstood the PR. I still would like the s, but you don't need the write.